.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "tools/modules/powershell/PSWriteColor"]
     path = tools/modules/powershell/PSWriteColor
     url = https://github.com/EvotecIT/PSWriteColor.git
+[submodule "vendor/configs/OpenColorIO-Configs"]
```

Comment:
@antirotor shouldn't we do this rather with poetry?

Reply:
@Allan-I look to `pyproject.toml` to line 34 

> acre = { git = "https://github.com/pypeclub/acre.git" }

Reply:
Thank you very much Jeza, I'm on it!

Reply:
Due to the package being in a format that poetry has problems with, and due to it giving me issues as I was grabbing it I've opted to keep including it as a git submodule unless further discussion arises, right now using poetry (according to what I understand from discussions online) might present future issues for that repository.

Reply:
I am only very afraid to do it as submodule as it is quite difficult to remove it lately..

Reply:
I absolutely agree with you on this also, it was a nightmare trying to remove it in the initial testing to see how it goes.  Also, it's quite a heavy repo actually.